packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -8,6 +8,7 @@
 export const AUDIO = 'audio';
 export const AUDIO_BANDWIDTH_LIMIT = '64000';
 export const AUDIOINPUT = 'audioinput';
+export const AUDIOSTATUS = 'audioStatus';
```

Comment:
what are the rules for casing here? are there any?

Reply:
none as of now heh